xdebug.c

```diff
@@ -1191,6 +1203,9 @@ PHP_RINIT_FUNCTION(xdebug)
 	XG(profiler_enabled) = 0;
 	XG(breakpoints_allowed) = 1;
 
+	/* Initialize gc statistics */
+	array_init(&XG(gc_runs));
```

Comment:
For almost every internal hash function, I use either xdebug_hash, or if possible, xdebug_list. As you can see in line 1207 (old) or 1222 (new) below.